test/fixtures/ignore.expected.css

```diff
@@ -28,6 +25,10 @@
 
 @import url(//css);
 
+@import "http://css" (min-width: 25em);
+
+@import "http://css-screen" (min-width: 25em) and screen;
+
```

Comment:
Why is the order changed here? This looks like a total weird side effect that will break user expected result...
If the ignored file is being imported first, imported content should be imported at this place, not later.
Looks like we are introducing a bug :)